pkgs/by-name/mi/mimikatz/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchzip
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "mimikatz";
+  version = "2.2.0-20220919";
+
+  src = fetchzip {
+    url = "https://github.com/gentilkiwi/mimikatz/releases/download/${finalAttrs.version}/mimikatz_trunk.zip";
+    hash = "sha256-wmatI/rEMziBdNiA3HE3MJ0ckdpvsD+LdbB7SKOYdI0=";
+    stripRoot=false;
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/mimikatz
+    cp -a * $out/share/mimikatz
```

Comment:
Also, could you install different binaries according to `stdenv.is64bit`?

Reply:
According to the usage on red teaming and pentesting activity, also `stdenv.is64bit` is not applicable here, because the pentester Linux could be 32-bit what the important aspect that the pentester knows is that the target Windows machine should be 64-bit.